src/AWS.Deploy.Recipes/RecipeDefinitions/BlazorWasm.recipe

```diff
@@ -166,29 +166,43 @@
                     "Name": "Uri",
                     "Description": "Uri to the backend rest api",
                     "Type": "String",
-                    "DefaultValue": "",
+                    "DefaultValue": "http://api-uri",
```

Comment:
I think we should have no default value for this field so the user has to enter a value. We want the toolkit to flag you must put a value here before publishing.

Reply:
To Norm's point, even if a user tries to deploy with an invalid value, the validators will all be run before we kick off deployments which should catch this https://github.com/aws/aws-dotnet-deploy/blob/e2ad9b5a026869aa8854182dd5df1f4cedc6fa46/src/AWS.Deploy.CLI/Commands/DeployCommand.cs#L723

Reply:
In general, for this https://github.com/aws/aws-dotnet-deploy/blob/main/src/AWS.Deploy.CLI/Commands/DeployCommand.cs#L834, I vote to remove that check. any objections @normj ?

Reply:
We can remove this check (If we decide to) as part of a separate PR.

Reply:
Also, setting invalid default values can easily trip customers as they could be performing deployments via a config file and may not specify an explicit value for all option settings

Reply:
But `"http://api-uri"` is likely invalid for the user's application that they're deploying, right? 

I don't think we should be defaulting a value just to get past client side validation, then deploy an application that likely won't work.

It feels like we have to have a concept/experience for _"this field is required even though there isn't a reasonable default value so you must provide one"_

Reply:
This needs a larger discussion.

Reply:
After syncing up with @normj, removed the default value on `BackendApi.Uri` and also removed  https://github.com/aws/aws-dotnet-deploy/blob/main/src/AWS.Deploy.CLI/Commands/DeployCommand.cs#L834